src/main/java/net/pms/util/FileNamePrettifier.java

```diff
@@ -0,0 +1,476 @@
+/*
+ * Universal Media Server, for streaming any media to DLNA
+ * compatible renderers based on the http://www.ps3mediaserver.org.
+ * Copyright (C) 2012 UMS developers.
+ *
+ * This program is a free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2
+ * of the License only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package net.pms.util;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.lang3.text.WordUtils;
+import net.pms.PMS;
+import net.pms.dlna.DLNAResource;
+import net.pms.dlna.RealFile;
+import net.pms.dlna.VideoClassification;
+
+
+/**
+ * This class "prettifies" video file names.
+ */
+public class FileNamePrettifier {
+
+	/** Strings that only occur after all useful information. */
+	public static final String COMMON_FILE_ENDS =
+		"[\\s\\.]AC3.*|[\\s\\.]REPACK.*|[\\s\\.]480p.*|[\\s\\.]720p.*|" +
+		"[\\s\\.]m-720p.*|[\\s\\.]900p.*|[\\s\\.]1080p.*|[\\s\\.]2160p.*|" +
+		"[\\s\\.]WEB-DL.*|[\\s\\.]HDTV.*|[\\s\\.]DSR.*|[\\s\\.]PDTV.*|" +
+		"[\\s\\.]WS.*|[\\s\\.]HQ.*|[\\s\\.]DVDRip.*|[\\s\\.]TVRiP.*|" +
+		"[\\s\\.]BDRip.*|[\\s\\.]BRRip.*|[\\s\\.]WEBRip.*|[\\s\\.]BluRay.*|" +
+		"[\\s\\.]Blu-ray.*|[\\s\\.]SUBBED.*|[\\s\\.]x264.*|" +
+		"[\\s\\.]Dual[\\s\\.]Audio.*|[\\s\\.]HSBS.*|[\\s\\.]H-SBS.*|" +
+		"[\\s\\.]RERiP.*|[\\s\\.]DIRFIX.*|[\\s\\.]READNFO.*|[\\s\\.]60FPS.*|" +
+		"[\\s\\.]Xvid.*|[\\s\\.]DivX.*|[\\s\\.]DVB.*";
+
+	/** A precompiled {@link Pattern} for {@link #COMMON_FILE_ENDS} */
+	protected static final Pattern COMMON_FILE_ENDS_PATTERN = Pattern.compile(COMMON_FILE_ENDS);
+
+	/**
+	 * Same as {@link #COMMON_FILE_ENDS}, but they are common words so reduce
+	 * the chances of false-positives by being case-sensitive.
+	 */
+	public static final String COMMON_FILE_ENDS_CASE_SENSITIVE =
+		"[\\s\\.]PROPER[\\s\\.].*|[\\s\\.]iNTERNAL[\\s\\.].*|" +
```

Comment:
For the record:
https://www.skyminds.net/les-differents-tags-dun-film-sur-internet/
https://scenenotice.org/details.php?id=2187

Reply:
@Sami32 For the record, I really can't be bothered with that 😛